Panels/tests/BreadcrumbDecorator-specs.js

```diff
@@ -1,192 +1,136 @@
-import PropTypes from 'prop-types';
-import kind from '@enact/core/kind';
-import {mount} from 'enzyme';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom';
+
 import BreadcrumbDecorator from '../BreadcrumbDecorator';
 import Panels from '../Panels';
 
-// 2019-04-11 - Skipped tests here are avoiding a Hooks testing issue. At this time, enzyme does not
-// properly test hooks, specifically the useCallback method.
-
 describe('BreadcrumbDecorator', () => {
 
-	const CustomBreadcrumb = kind({
-		name: 'CustomBreadcrumb',
-
-		propsTypes: {
-			index: PropTypes.number,
-			onSelect: PropTypes.func
-		},
-
-		render: ({index, onSelect}) => {
-			const handleSelect = () => onSelect({index});
-			return <span onClick={handleSelect}>{index}</span>;
-		}
-	});
-
 	const Panel = () => <div />;
 
-	test.skip('should wrap primitive breadcrumbs with Breadcrumb', () => {
+	test('should wrap primitive breadcrumbs with Breadcrumb', () => {
 		const SingleBreadcrumbPanels = BreadcrumbDecorator({
 			max: 1
 		}, Panels);
 
-		const subject = mount(
+		render(
 			<SingleBreadcrumbPanels index={2} breadcrumbs={['1st', '2nd', '3rd']}>
 				<Panel />
 				<Panel />
 				<Panel />
 			</SingleBreadcrumbPanels>
 		);
 
-		const expected = '2nd';
-		const actual = subject.find('Breadcrumb').text();
+		const actual = screen.getByText('2nd');
 
-		expect(actual).toBe(expected);
+		expect(actual).toBeInTheDocument();
 	});
 
-	test.skip('should support custom breadcrumbs', () => {
-		const SingleBreadcrumbPanels = BreadcrumbDecorator({
-			max: 1
-		}, Panels);
-
-		const breadcrumbs = [0, 1, 2].map(i => <CustomBreadcrumb index={i} />);
-
-		const subject = mount(
-			<SingleBreadcrumbPanels index={2} breadcrumbs={breadcrumbs}>
-				<Panel />
-				<Panel />
-				<Panel />
-			</SingleBreadcrumbPanels>
-		);
-
-		const expected = 1;
-		const actual = subject.find('CustomBreadcrumb').length;
-
-		expect(actual).toBe(expected);
-	});
```

Comment:
Note to reviewer: This test was deleted beacause there is no indication in the DOM that we are using CustomBreadcrumbs instead of common Breadcrumbs.